heron/executor/src/python/heron_executor.py

```diff
@@ -192,6 +201,45 @@ def __init__(self, args, shell_env):
 
     self.state_managers = []
 
+  @staticmethod
+  def parse_args(args):
+    """Uses python argparse to collect positional args"""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("shard")
```

Comment:
elsewhere inthe code we use `--shard` instead. What's the difference in usage when using `shard`?